src/pages/dashboard/newpaint/NewPainting.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class="row q-mt-xl">
+
+    <div class="col-xs-12 col-sm-6 items-center ">
+      <div class=" q-col-gutter-md row justify-center items-center ">
+        <q-img
+          src="../../../assets/images/hashly-gwei.svg"
+          style="height: 429px; width: 230px; border:none;">
+        </q-img>
+      </div>
+    </div>
+
+    <div class="col-xs-12 col-sm-6 ">
+      <div class="header">
+        {{ $t('dashboard.newpainting.create') }}
+      </div>
+
+      <div class="q-mt-lg">
+        <label>{{ $t('dashboard.newpainting.TypeAText') }}</label>
+        <input v-bind:maxlength="limit"
+        v-model="textType" required class="full-width typeText"/>
+        <p class="justify-end text-right">
+          {{typed}}{{ $t('dashboard.newpainting.trace') }}{{remains}}
+        </p>
+        <div class="col-xs-6 col-sm-3 ">
+          <span>{{ $t('dashboard.newpainting.PaintOnWall') }}</span>
+          <div class="row">
+            <q-radio  v-model="hang" val="Yes" label="Yes"></q-radio>
+            <q-radio  v-model="hang" val="No" label="No"></q-radio>
+          </div>
+        </div>
+
+        <div>
+          <span>{{ $t('dashboard.newpainting.CoresAleatórias') }}</span>
+          <div class="row">
+            <q-radio v-model="range" val="yes" label="Yes"></q-radio>
+            <q-radio v-model="range" val="no" label="No"></q-radio>
+          </div>
+          <div v-if="range==='yes'" class="q-mt-md" >
+            <p>{{ $t('dashboard.newpainting.CollorInversionProbability') }}</p>
+            <q-slider v-model="slider" color="primary" :min="0" :max="50"/>
+          </div>
+        </div>
+
+        <div class="q-mt-md">
+          <p>{{ $t('dashboard.newpainting.YourInspiration') }}</p>
+          <div class="row">
+            <div class="col-xs-12 col-sm-6 items-center">
+              <div class="col">
+                <q-radio  v-model="test" val="Random" label="Random"></q-radio>
+              </div>
+              <div class="col">
+                <q-radio  v-model="test" val="Calm" label="Calm"></q-radio>
+              </div>
+              <div class="col">
+                <q-radio  v-model="test"
+                val="Colorfull blocks" label="Colorfull blocks"/>
```

Comment:
```suggestion
                <q-radio
                   v-model="test"
                  val="Colorfull blocks"
                  label="Colorfull blocks"
                />
```
Colocar valor do label no arquivo de mensagens.